external/storm-hdfs/src/test/java/org/apache/storm/hdfs/bolt/AvroGenericRecordBoltTest.java

```diff
@@ -72,24 +73,17 @@ public class AvroGenericRecordBoltTest {
     private static Schema schema2;
     private static Tuple tuple1;
     private static Tuple tuple2;
-    @Rule
-    public MiniDFSClusterRule dfsClusterRule = new MiniDFSClusterRule(() -> {
-        Configuration conf = new Configuration();
-        conf.set("fs.trash.interval", "10");
-        conf.setBoolean("dfs.permissions", true);
-        File baseDir = new File("./target/hdfs/").getAbsoluteFile();
-        FileUtil.fullyDelete(baseDir);
-        conf.set(MiniDFSCluster.HDFS_MINIDFS_BASEDIR, baseDir.getAbsolutePath());
-        return conf;
-    });
+
+    @RegisterExtension
+    public static final MiniDFSClusterExtension DFS_CLUSTER_EXTENSION = new MiniDFSClusterExtension();
```

Comment:
This needs a config supplier as parameter to constructor (with config values as specified in the Rule)

Reply:
done